### DISCUSSION TITLE
Reload of development browser odd issue?

### DISCUSSION THREAD

**Discussion by linoxyn** [raae/nattermob.dev#35] [Q&A] [answered]
Hi pirates! 

I'm not sure what the issue may be when I'm working in development - I played with code, and in the `live-now-marquee.js` file, I got something happening (sorry my coding is maybe not that great) and when I save the file and don't see a change, or I just hard refresh the browser, some of the function I added does not work as expected. The 'crazy' marquee colour changes stops. 

To see what I tried to code, here is my forked repo: https://github.com/linoxyn/nattermob.dev

I didn't attempt a **Pull Request** as the marquee is still not on the table. 
Thank you for the fun!

**Comment by raae** [comment]
Hey! 

Love that you took it for a spin. I'll have a look ~~tonight~~ tomorrow morning 👀

/ Queen Raae

**Comment by raae** [comment, marked as answer]
I went trough the code and managed to get it working. Not sure exactly what made it click (other than probably something to do with (useEffect/useCallback), but I tried to do a commit at each step so you can follow along. If you give me access to your repo I can push my branch there and you can have look.
